gateway/loaders/zookeeper/utils.go

```diff
@@ -17,6 +22,55 @@ type ZookeeperClient struct {
 	conn *zk.Conn
 }
 
+func (z *ZookeeperTargetLoader) refreshZookeeperConfig(log zerolog.Logger, targetChan chan<- *connections.TargetConnectionControl, config *configuration.GatewayConfig) error {
```

Comment:
nit: the method name is a bit confusing. The target config is also stored in zookeeper. How about "refreshGlobalConfig"?